dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -562,49 +562,57 @@ CDefaultContextMenu::AddStaticContextMenusToMenu(
         /* By default use verb for menu item name */
         mii.dwTypeData = pEntry->szVerb;
 
+        WCHAR wszKey[256];
+        StringCbPrintfW(wszKey, sizeof(wszKey), L"shell\\%s", pEntry->szVerb);
```

Comment:
What about checking for success/failure? (as was done before)

Reply:
Added check.